.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 ###########
 
 # Catch all for whole project
-* @CIPop @danewalton @ericwol-msft @ewertons @jspaith @josesanchez7 @preethi826 @vaavva
+* @CIPop @danewalton @ericwol-msft @ewertons @jspaith @josesanchez7 @preethi826 @RLeclair @vaavva
```

Comment:
sorry while you're here can you fix my error and put jose before jspaith? I don't know my abc's 😬